dom/schema/v1/tollgate.sql

```diff
@@ -0,0 +1,9 @@
+create table if not exists tollgate
+(
+    id         text        not null primary key,
+    name       text        not null,
+    b_boxes    jsonb,
+    gate_line  jsonb,
+    created_at timestamptz not null,
+    updated_at timestamptz
```

Comment:
I suggest we don't use redundant '_at' suffix everywhere. Just `created`, `updated` are perfect. More unnecessary typing later.

Reply:
agree